test/common.js

```diff
@@ -21,6 +21,35 @@ exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
 var opensslCli = null;
 
+Object.defineProperty(exports, 'inFreeBSDJail', {
```

Comment:
Perhaps the return value of these should be cached? In test-net-local-address-port.js already you're using `localhost_ipv4` twice and it's easy to forget that dynamic properties are lazy. See `opensslCli` for an already cached property.